### ISSUE TITLE
Links missing from website

### ISSUE THREAD

**Issue by sebbASF** [apache/incubator-skywalking#1031]
There do not appear to be any links to the following on the website:

- issue tracker
- mailing lists

ASF projects must also link to:
- AL 2.0 license
- Sponsorship/Donate
- Thanks
See: https://www.apache.org/foundation/marks/pmcs#navigation

**Comment by wu-sheng** [comment]
Thanks, I will adjust our website.

**Comment by wu-sheng** [comment]
Links updated.

![image](https://user-images.githubusercontent.com/5441976/38310536-61f415be-384f-11e8-9a33-b386549b1562.png)

**Comment by wu-sheng** [comment]
@sebbASF Are the links good now?

**Comment by sebbASF** [comment]
Yes and no. The links are present, but they are expected to be part of the menu system so they are more visible. At present they are not all that easy to find, as the menu item "Links" suggests 3rd party URLs.

If you have a look at some other ASF pages, you will see the difference.

**Comment by sebbASF** [comment]
It does not have to be a sub-level, though that is a common approach.
Many other sites use a side menu.
